orderbook/src/u256_conversions.rs

```diff
@@ -0,0 +1,30 @@
+use bigdecimal::BigDecimal;
+use num_bigint::{BigInt, Sign, ToBigInt};
+use primitive_types::U256;
+
+pub fn u256_to_big_int(input: &U256) -> BigInt {
```

Comment:
I have used this also in the less naive solver. Maybe we can create a util crate to collect common helper functions like this.